app/components/pages/Login/index.js

```diff
@@ -17,6 +20,12 @@ const EXCHANGE_TOKEN_MUTATION = gql`
     exchangeJournalToken(token: $token)
   }
 `
+const Container = styled(Box)`
+  padding-top: ${th('space.3')};
+  ${media.tabletPortraitUp`
+    padding-top:${'calc(50vh - 300px)'};
```

Comment:
The AppBar takes up a total of 144px of space on the screen at desktop (71px tall, 1px border, 72px bottom-margin). This isn't being included in this calc which is causing the content to be pushed too far down.

Reply:
@Cooryd I don't think the bottom margin on the AppBar comes into play here because of margin collapsing? The `margin-top` on this `Box` is larger than the `margin-bottom` on the AppBar, so the margin-top on the Box will win out. Right?

Our margin-top comes from this calculation:
(`height of viewport` - `height of app bar` - `height of Box`) / `2`
(`100vh` - `72px` - `548px`) / 2
which can be simplified to: `50vh - 310px`

I think 300px rather than 310 may just have been a rough estimate, or rounding?

Reply:
![screenshot 2018-11-05 at 13 04 36](https://user-images.githubusercontent.com/5822727/47997228-6b839a00-e0fb-11e8-9d97-5f7f4ed30b7f.png)
![screenshot 2018-11-05 at 13 04 15](https://user-images.githubusercontent.com/5822727/47997231-6d4d5d80-e0fb-11e8-97c2-62582c2c8a50.png)